### ISSUE TITLE
Package names are not descriptive: "package com.github.pedrovgs.problem65"

### ISSUE THREAD

**Issue by FractalizeR** [pedrovgs/Algorithms#1]
I guess package names should not contain anything as meaningless as "problem65". The greatest virtue of today's world is a code reuse. I would like to import code from something with a descriptive name so as not to confuse my followers.

Why isn't it called  `package com.github.pedrovgs.algorithms.IsTreeBalanced`?

**Comment by pedrovgs** [comment]
This repo is not going to be published to any artefacts repository like Maven Central. This repository contains just some algorithms I wrote to practice and is a reference to read, not a library to consume. You have this algorithms categorized inside the README.md. 

To me, in this repository the package name is not important, I use this structure just to keep the reference of the number of problems I've solved and the order I've used. If this was a library published and ready to be used I understand your point, but this is like a reference book where we can break some rules :)
